### DISCUSSION TITLE
Learnable parameters for mean prior.

### DISCUSSION THREAD

**Discussion by lovecodng** [cornellius-gp/gpytorch#2275] [Q&A] [answered]
I am wondering if its possible to fit the following model using multitask GP, where all hyper-parameters should be learned by the model.

![Screenshot 2023-02-14 213206](https://user-images.githubusercontent.com/72666278/218826575-c1ee791e-a4f8-431a-84fa-0b844676c55d.jpg)

**Comment by gpleiss** [comment, marked as answer]
This should be possible since y1 and y2 are jointly normal.

Since `y1 | y2 = N(const + a1 y2 - b1 ln(x) | k_1)`, we have that `y1 = N(const + a1(const - b2 ln(x)) - b1 ln(x) | k1 + k2 + (const - b2 ln(x))^2)`.

You can implement a MultitaskMean function that parameterizes the mean in this way.
